Zeeguu Reader/Zeeguu Reader/ZGJavaScriptAction.swift

```diff
@@ -117,22 +117,29 @@ enum ZGJavaScriptAction {
 	func getJavaScriptExpression() -> String {
 		switch self {
 		case let .Translate(dict):
-			guard let word = dict["translation"], context = dict["context"], id = dict["id"], bid = dict["bookmarkID"] else {
+			guard let translation = dict["translation"], word = dict["word"], context = dict["context"], id = dict["id"], bid = dict["bookmarkID"] else {
 				fatalError("The ZGJavaScriptAction.Translate(_) dictionary is in an incorrect state!")
 			}
-			var w = word.stringByReplacingOccurrencesOfString("\\", withString: "\\\\")
-			w = w.stringByReplacingOccurrencesOfString("\"", withString: "\\\"")
+			var t = translation.stringByReplacingOccurrencesOfString("\\", withString: "\\\\")
+			t = t.stringByReplacingOccurrencesOfString("\"", withString: "\\\"")
+			t = t.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet())
 			
 			var c = context.stringByReplacingOccurrencesOfString("\\", withString: "\\\\")
 			c = c.stringByReplacingOccurrencesOfString("\"", withString: "\\\"")
+			c = c.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet())
+			
+			var w = word.stringByReplacingOccurrencesOfString("\\", withString: "\\\\")
+			w = w.stringByReplacingOccurrencesOfString("\"", withString: "\\\"")
+			w = w.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet())
```

Comment:
you're repeating yourself mister. why not define an `escapeSpecialChars` function or something like that? also which would have a comment on why this is important.